bin/ns-delete-old-devicestatus.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+
+source $(dirname $0)/oref0-bash-common-functions.sh || (echo "ERROR: Failed to run oref0-bash-common-functions.sh. Is oref0 correctly installed?"; exit 1)
+
+usage "$@" <<EOF
+Usage: $self --find <NIGHTSCOUT_HOST> <asn1 of API_SECREAT>- No-op version, find out what delete would do.
+$self delete <NIGHTSCOUT_HOST>  <asn1 of API_SECREAT>- Delete duplicate entries from ${NIGHTSCOUT_HOST-<NIGHTSCOUT_HOST>}
```

Comment:
API_SECREAT should be API_SECRET.  What exactly is `asn1`?

Reply:
Obviously, I meant to say sha1.

The reason that I wanted to give it as the sha1 and not the api_secreat itself, is because as far as I have noticed, openaps only stores the sha1 of the api secret.

My long term result is to call this automatically, and thus stop the rigs from failing after a few months of good work.